hedera-node/hedera-app/src/xtest/java/contract/ScaffoldingModule.java

```diff
@@ -179,21 +184,20 @@ static SavepointStackImpl provideSavepointStackImpl(
     @Singleton
     static Function<TransactionBody, HandleContext> provideHandleContextCreator(
             @NonNull final Metrics metrics,
-            @NonNull final HederaState state,
+            @NonNull final NetworkInfo networkInfo,
             @NonNull final RecordCache recordCache,
             @NonNull final Configuration configuration,
             @NonNull final ConfigProvider configProvider,
             @NonNull final ServiceScopeLookup scopeLookup,
             @NonNull final BlockRecordManager blockRecordManager,
-            @NonNull final WorkingStateAccessor stateAccessor,
             @NonNull final TransactionDispatcher dispatcher,
             @NonNull final SavepointStackImpl stack) {
-        stateAccessor.setHederaState(state);
         final var parentRecordBuilder = new SingleTransactionRecordBuilderImpl(Instant.now());
```

Comment:
How can `Instant.now()` be the consensus timestamp (the javadoc for the argument)?

Reply:
As above.